features/FxA-54-connect-another-device-phase-3/README.md

```diff
@@ -0,0 +1,367 @@
+5. The Google Play store will open. Click the `Install` button.
+6. Firefox should install. When complete, click `Open`.
+7.  Firefox will open to the Firefox for Android firstrun page
+6. Open Firefox, open `https://<server>/signin?context=fx_fennec_v1&service=sync&signin=<signinCode from step 4>`
+7. When opening Firefox Accounts, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+###### Sign up in Firefox desktop, send link to Android, Firefox already installed.
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device. The link will be of the form `https://<server>/m/:signinCode`, write down the `signinCode`.
+5. Firefox should open.
+6. Open a tab to `https://<server>/signin?context=fx_fennec_v1&service=sync&signin=<signinCode from step 4>`
+7. When opening Firefox Accounts, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+##### Testing with support for deep links in Firefox for Android
+
+###### Sign up in Firefox desktop, send link to Android, Firefox not installed
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device.
+5. Install Firefox from the Google Play store.
+6. Open Firefox, complete the "firstrun flow".
+7. Firefox Accounts should open, email address from step 1 should be filled in.
+8. Sign in with the password from step 1.
+
+###### Sign up in Firefox desktop, send link to Android, Firefox already installed.
+
+1. Using Firefox desktop, sign in to Sync by creating a new Firefox Account using
+   a mozilla or softvision address.
+2. In the same instance of Firefox desktop, open the verification link with
+   the `&signinCodes=true&forceExperimentGroup=treatment` query parameters.
+3. Enter a valid phone number for an Android device, submit.
+4. Open the SMS link on an Android device.
+5. Firefox should open to the firstrun flow. Complete the "firstrun flow".
+6. Firefox Accounts should open, email address from step 1 should be filled in.
+7. Sign in with the password from step 1.
+
+##### Reference Images
+
+###### Send SMS
+<img src="images/send_sms.png" width="250" alt="Send SMS"/>
+
+###### SMS received
+Image needed
+
+###### Install Firefox
+<img src="images/install_firefox_play_store_android.png" width="250" alt="Send SMS"/>
+
+###### Open Firefox
```

Comment:
@ryanfeeley should have new screenshots for when users open app from deeplink. The onboarding flow order on iOS should be changed from deeplinks to put FxA first (rather than last) since the user is completing a pairing flow.